howso/details.amlg

```diff
@@ -89,6 +89,15 @@
 	;						locally around the prediction. Uses only the context features of the reacted case to
 	;						determine that area. Relies on 'robust_residuals' flag.
 	;
+	;		 "prediction_stats" true or false. If true outputs feature prediction stats for all (context and action) 
+	;						features locally around the prediction. The stats returned are ("r2", "rmse", "spearman_coeff",
+	;						"precision", "recall", "accuracy", "mcc"). The "confusion_matrix" may also be returned by setting
+	;						'confusion_matrix' to true. Uses only the context features of the reacted case to determine that 
+	;						area. Relies on 'robust_residuals' flag.
+	;
+	;		 "confusion_matrix" true or false. If true, will automatically set 'prediction_stats' to true and return the 
```

Comment:
"the confusion matrix".. 

This makes it sound like there would only be one. There should be one for each nominal feature, right?

Reply:
yes